README.md

```diff
@@ -54,6 +54,8 @@ JSON properties are written in the following format: `"key" [quantifier]: valueT
       "eventName": String,
       "startDate": unix_time,
```

Comment:
Did you mean to keep both appearances of `startDate` and `endDate` in lines 55/57 and 56/58, respectively?